### ISSUE TITLE
[Visual requirement - Make code - Share project]: Luminosity ratio of the keyboard focus indicator on the 'X' (Close) button does not meet minimum contrast ratio 3:1.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#5467]
### User Experience 
Low vision people will not be able to find the focused control if luminosity ratio doesn't meet the minimum requirement of 3:1. Due to which the users will not be able to close the project dialog.

### Note: 
User credentials should NOT be included in the bug. 

### Repro Steps:  

1. Launch application URL: https://makecode.microbit.org
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate through all the controls of  "Share Project" dialog.

### Actual Result:
 Luminosity ratio of the keyboard focus indicator on the 'X' (Close) button does not meet minimum contrast ratio 3:1.

**Observation:** The actual luminosity ratio of the focus indicator on 'X' button is 2.9:1.

### Expected result:
 Luminosity ratio of the keyboard focus indicator on the 'X' (Close) button should be greater than or eqaul to minimum contrast ratio 3:1.

### MAS Reference:
[MAS 1.4.11 – Non-text Contrast](https://aka.ms/MAS1.4.11)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance: 
NA

### DOM details available:
NA

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDc3NDY0MzExZDkxNTRlYmRhNzk2YTExYzE5Y2I2N2ZifDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3ODcwNDg4MTQ1NjM2NzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT0%3D&sdata=VHBCSFBzMTdmWExVNTk3MzZlTHdiME4rcFo3d3lyYk5kTTIvRjhLdTN3QT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-chbhardwaj%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.

### Test Environment:

- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- Tool: Color Contrast Analyser
- URL: https://makecode.microbit.org.
<img width="1292" alt="MAS1 4 11_Luminosity ratio of the keyboard focus indicator on the 'X' (Close) button does not meet minimum contrast ratio 3 1" src="https://github.com/microsoft/pxt-microbit/assets/158465724/b2dbcf11-3162-4543-a708-5453f6dfdfda">

**Comment by vroopea** [comment]
#Accessibility;#A11yMAS;#A11ySev3;#A11yWCAG2.1;#MAS1.4.11;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Jan2024;
